lib/RTCSession.js

```diff
@@ -1036,15 +959,7 @@ module.exports = class RTCSession extends EventEmitter
     if (transportType === JsSIP_C.DTMF_TRANSPORT.RFC2833)
     {
       // Send DTMF in current audio RTP stream.
-      const sender = this._getDTMFRTPSender();
-
-      if (sender)
-      {
-        // Add remaining buffered tones.
-        tones = sender.toneBuffer + tones;
-        // Insert tones.
-        sender.insertDTMF(tones, duration, interToneGap);
-      }
+      this._connection.sendInbandDTMF(tones, duration, interToneGap);
```

Comment:
Please, test that this method is working as it was.

```suggestion
      this._connection.sendDTMF(tones, duration, interToneGap);
```